src/python/install.sh

```diff
@@ -454,6 +454,19 @@ if [[ "${INSTALL_PYTHON_TOOLS}" = "true" ]] && [[ $(python --version) != "" ]];
             echo "${util} already installed. Skipping."
         fi
     done
+    
+    # Temporary: Removes “setup tools” metadata directory due to https://github.com/advisories/GHSA-r9hx-vwmv-q579
+
+    VULNERABLE_VERSIONS=("3.10" "3.11")
+
+    for vv in "${VULNERABLE_VERSIONS[@]}"; do
+
+        if [ "${PYTHON_VERSION}" == "${vv}" ]; then
+            rm -rf  ${PIPX_HOME}/shared/lib/"python${vv}"/site-packages/setuptools-65.5.0.dist-info
+            #rm -rf ${PYTHON_INSTALL_PATH}/"${VERSION}"/lib/python${vv}/site-packages/setuptools-65.5.0.dist-info
```

Comment:
Can you remove this comment?

Reply:
I will remove that but I would like to highlight that sometime another directory will get created for setuptools-65.5.0 do do you want us to remove that or not?

Reply:
> highlight that sometime another directory will get created for setuptools-65.5.0

Do you have a scenario or reproduction for this? Are you aware of the drawbacks of removing that directory? 

We shouldn't add code or comments if it's not 💯 needed. However, if the scanners complain about it, then we can definitely add it back.

Reply:
I have implemented a way to dynamically fetch the python version so now no issue.